ext/standard/tests/directory/DirectoryClass_cannot_clone.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Cannot serialize instance of Directory class constructed via Reflection.
+--FILE--
+<?php
+
+$d = dir(__DIR__);
+try {
+    $cloned = clone $d;
+    $cloned_files = [];
```

Comment:
Given that the error is going to come from the `clone` call on line 8 as far as I understand, why include the rest of the `try` block? It shouldn't be executed

Reply:
Same remark.